galaxyui/src/app/resources/survey/survey.service.ts

```diff
@@ -1,59 +1,14 @@
-import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 
-import { Observable, of } from 'rxjs';
-import { catchError, map, tap } from 'rxjs/operators';
-
 import { NotificationService } from 'patternfly-ng/notification/notification-service/notification.service';
-import { PagedResponse } from '../paged-response';
 import { Survey } from './survey';
 
-const httpOptions = {
-    headers: new HttpHeaders({
-        'Content-Type': 'application/json',
-    }),
-};
-@Injectable()
-export class SurveyService {
-    private url = '/api/v1/community_surveys';
-
-    constructor(private http: HttpClient, private notificationService: NotificationService) {}
-
-    query(params?: any): Observable<Survey[]> {
-        params.page_size = 1000;
```

Comment:
I don't see a default page_size represented in the generic class. Is this going to create a problem? Setting a page size of 1000 was a lazy way around having to handle paging. The default API page size is 10, so if more than 10 items exist, and the page size is not set to a value greater than the total number of items , the query method would need to recursively call the API to retrieve all pages.

Reply:
It shouldn't cause problems because I moved the `page_size` param here: https://github.com/ansible/galaxy/pull/1131/files#diff-f186fcc519707d031dbbd1eae2d89ba4R93

Reply:
Is there a better way to load all of the surveys than set the page size to 1000?